src/blob/index/benchmarks.rs

```diff
@@ -32,14 +32,14 @@ impl Default for KeyType {
 type FileIndexStruct = BPTreeFileIndex<KeyType>;
 //type FileIndexStruct = SimpleFileIndex;
 
-fn generate_headers(records_amount: usize, key_mapper: fn(u32) -> u32) -> InMemoryIndex {
-    let mut inmem = InMemoryIndex::new();
+fn generate_headers(records_amount: usize, key_mapper: fn(u32) -> u32) -> InMemoryIndex<KeyType> {
+    let mut inmem = InMemoryIndex::<KeyType>::new();
     (0..records_amount as u32)
         .map(key_mapper)
         .map(|i| serialize(&i).expect("can't serialize"))
-        .for_each(|mut key| {
-            key.resize(KeyType::LEN as usize, 0);
-            let rh = RecordHeader::new(key.clone(), 1, 1, 1);
+        .for_each(|key| {
+            let key: KeyType = key.into();
+            let rh = RecordHeader::new(key.clone().to_vec(), 1, 1, 1);
```

Comment:
Do we need to `clone` the key if it is immediately converted to a vector?

Reply:
This code left from the experiments with consuming `to_vec`. Fixed in all cases